### PR TITLE
Update version to reflect the latest patch release

### PR DESCRIPTION
**Pull request by rexagod**
Reflect patch release changes on `main`, which was not the case with `2.10.1` since that diverged and was never merged back into `main` (around backporting concerns). I believe in the future we can merge everything back into `main` to reflect the latest version changes in main, but diverge the release branches starting from the pre-merge commit to make sure backports are clean and maintainable (these will not be merged back into the default branch).

### CODE REVIEW

File: tools/tools.go [hunk 1]
```diff
@@ -20,6 +20,7 @@ limitations under the License.
 package tools
 
 import (
+	_ "github.com/Kunde21/markdownfmt/v3"
```
Comment (dgrisonnet) [inline tools/tools.go:23]:
```suggestion
	_ "github.com/Kunde21/markdownfmt/v3/cmd/markdownfmt"
```

**Comment by rexagod** [comment]
/close
Superseded by #2317.

**Pull request closed without merging**